rapids_pytest_benchmark/rapids_pytest_benchmark/plugin.py

```diff
@@ -249,10 +248,11 @@ def runner():
                     time.sleep(0.1)
 
             finally:
-                stopGpuMetricPolling(gpuPollObj)
+                rmm_analyzer.disable_logging()
+                rmm_analyzer.parse_results()
```

Comment:
I noticed the log that's parsed in this method is suffixed with the device ID.  I wonder if we need to keep the device number (as was used in the previous version) in place and pass it to this method so the right log can be parsed?